docs/design/admission_control_limit_range.md

```diff
@@ -0,0 +1,122 @@
+# Admission control plugin: LimitRanger
+
+## Background
+
+This document proposes a system for enforcing min/max limits per resource as part of admission control.
```

Comment:
I'd like to see a user-documentation version of this, in addition to, or in place of this "docs/design" file.  It would be pretty close to this, but with phrases like "is introduced" and "future enhancements" removed.  Assuming you do that in a separate PR.  You may want to merge all the admission controller user docs into one file for readability.  Up to you.